lib/pos_epoch.go

```diff
@@ -14,7 +14,10 @@ import (
 
 type EpochEntry struct {
 	EpochNumber      uint64
-	FinalBlockHeight uint64
+	StartBlockHeight uint64
+	StartViewNumber  uint64
+	FinalBlockHeight uint64 // TODO: I'm leaving this here currently because it makes our lives easier
```

Comment:
Ah yeah, we definitely need FinalBlockHeight. Very convenient to leave it as-is
```suggestion
	FinalBlockHeight uint64
```